packages/react-scripts/template/README.md

```diff
@@ -430,8 +429,8 @@ Then we can change `start` and `build` scripts to include the CSS preprocessor c
 
 ```diff
    "scripts": {
-     "build-css": "node-sass src/ -o src/",
-     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
+     "build-css": "node-sass-chokidar src/ -o src/",
+     "watch-css": "npm run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
```

Comment:
Thank to your reply. I mean that I want to use "node-sass-chokidar" but still can import .scss files.
For example: my Home folder includes: Home.js, Home.scss, ... 
After run "npm start", my folder add Home.css. So I'm scared of that one day I forget and edit css files instead of scss.
Another issue is that it's assume that Styles folder has 10 SCSS files, and now after run "npm start" it will have 20 files. I knew we can ignore CSS files, but it makes structure of app folders cumbrous.

In some react apps built with webpack 2 as I did, we need just import .scss files, can run "npm start" in dev mode without compiling new css files. Therefore, can you improve it?